### PR TITLE
Mbroz readme update4

### PR DESCRIPTION
**Pull request by mbroz2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
 # Contributing to the blogs
 
 1. Create an issue for your blog post in this repository.
-2. Fork this repository and clone.  Note that you're working off of the default `prod` branch.
+2. Clone the repo and create your feature branch off of the default `prod` branch.  Alternatively, you can fork this repo and create your personal branch in your fork, but this may make collaboration with others, including reviewers like @lauracowen, more problematic for some in certain situations.
```
Comment (steven1046) [inline README.md]: "for some in certain situations" sounds confusing.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
+11. Create a PR from `staging` branch into `prod` branch (with an admin approver, though this check is currently purposefully disabled).  Once approved, merge into `prod`.
+12. Rebuild the [production site](https://openliberty.io/blog/) on IBM Cloud and verify the post looks as expected on openliberty.io.  If the post used a future date, it will not render on the production site until that day or later (and will need the site to be rebuild to publish).
+13. Now that the contents of the feature branch has been successfully delivered to & tested on `draft`, `staging`, and `prod`, delete the feature branch.
 
 Summary of commit flow through branches:  
-`prod` <-PR- `staging` <-PR- `draft` <-PR- `personal branch from fork`  
-In other words, `prod` should always be a commit subset of `staging` which should be a commit subset of `draft`.  Note, however, that you're not actually, normally, doing a PR from `draft` to `staging` but instead from your `personal branch from fork` into `staging`, but only after it already has been in `draft`.
+`prod` <-PR- `staging` <-PR- `draft` <-PR- `feature/personal branch from fork`
```
Comment (steven1046) [inline README.md]: Can we make it more clear that we shouldn't make pr's from draft into staging into prod? Maybe something like:
feature/personal branch -> draft
test on draft site
feature/personal branch -> staging
final testing and Laura's/SME's review
feature/personal branch -> prod
Build Ol.io

**Review by steven1046 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mbroz2) [reply README.md]: Yes, this section does need a better flow diagram and I might just remove it in the interim, but as a side note, I would expect a PR from staging into prod, and not from the feature/personal branch into prod.  I think prod should only ever have PRs from staging, with the exception being some extremely time-sensitive fix of something that's already live on the prod site.

Comment (steven1046) [reply README.md]: That's true about staging->prod

**Review by steven1046 — APPROVED** [review]
